UnitsNet/UnitDefinitions/Volume.json

```diff
@@ -434,6 +434,30 @@
           "Abbreviations": [ "bbl" ]
         }
       ]
+    },
+    {
+      "SingularName": "UsBeerBarrel",
+      "PluralName": "UsBeerBarrels",
+      "FromUnitToBaseFunc": "x*0.1173477658",
+      "FromBaseToUnitFunc": "x/0.1173477658",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "bl (U.S.)" ]
```

Comment:
Won't this abbreviation conflict with fluid and dry barrels? If we have ambiguous units, then Volume.Parse("1 bl (U.S)") would fail with an exception. It is very possible that some units actually do have ambiguous abbreviations and in that case it is correct to not allow to parse it. Just checking in as I can't easily find the abbreviations for fluid and dry barrels. I just see there are a lot of different kinds of barrels mentioned everywhere.

Reply:
http://convert-to.com/conversion/volume/volume_multimenu.html

I have usually seem fluid barrels as fl bl